tests/waku_rln_relay/test_waku_rln_relay.nim

```diff
@@ -710,6 +705,55 @@ suite "Waku rln relay":
       msgValidate3 == MessageValidationResult.Valid
       msgValidate4 == MessageValidationResult.Invalid
 
+  asyncTest "validateMessageAndUpdateLog: multiple senders with same external nullifier":
+    let index1 = MembershipIndex(5)
+    let index2 = MembershipIndex(6)
+
+    let rlnConf1 = WakuRlnConfig(rlnRelayDynamic: false,
+                                 rlnRelayCredIndex: some(index1),
+                                 rlnEpochSizeSec: 1,
+                                 rlnRelayTreePath: genTempPath("rln_tree", "waku_rln_relay_3"))
+    let wakuRlnRelay1Res = await WakuRlnRelay.new(rlnConf1)
+    if wakuRlnRelay1Res.isErr(): assert false, "failed to create waku rln relay: " & $wakuRlnRelay1Res.error
+    let wakuRlnRelay1 = wakuRlnRelay1Res.get()
```

Comment:
The `valueOr` and `isOkOr` stew templates are very handy. In this case, we could have:
```suggestion
    let wakuRlnRelay1 = (await WakuRlnRelay.new(rlnConf1)).valueOr:
      assert false, "failed to create waku rln relay: " & $error
```